libs/community/langchain_community/output_parsers/pandas_dataframe.py

```diff
@@ -0,0 +1,176 @@
+import re
```

Comment:
lets keep this in langchain for now.

Reply:
Nah. I've made a research before moving pandas parser. We have the pandas data_loader in community package and nothing in core. The only pandas dependent piece in langchain was this parser. That's why.